build_scripts/build_wheel.sh

```diff
@@ -0,0 +1,55 @@
+#bin/bash
+
+# Ensure that 1 argument (ROOT_DIR) is provided
+die () {
+    echo >&2 "$@"
+    exit 1
+}
+[ "$#" -eq 1 ] || die "1 argument required, $# provided"
+ROOT_DIR=$1
+
+# This needs to match the version in main on Github
+THESEUS_VERSION=$(head theseus/__init__.py -n 6 | tail -n 1 | awk -F'"' '{print $2}')
+
+for PYTHON_VERSION in 3.9; do
```

Comment:
Maybe we have a file with the list of supported versions for python, cuda, etc where they can be pulled from instead of hardcoded in this script?

Reply:
We could add them to `theseus/__init__.py`.

Reply:
Not sure if this is necessary, since we are currently not using this information in any other script. At least not in a way where simply adding a new version to this file would automatically change some workflow.